examples/GainPlugin/Source/jsui/webpack.config.js

```diff
@@ -2,8 +2,12 @@ module.exports = {
   entry: './src/index.js',
   output: {
     path: __dirname + '/build/js',
-    filename: 'main.js'
+    filename: 'main.js',
+    sourceMapFilename: "[file].map",
+     devtoolModuleFilenameTemplate: info =>
+    `webpack:///${info.absoluteResourcePath.replace(/\\/g, '/')}`
```

Comment:
Required to make vscode debug extension on Windows behave. See: https://github.com/harold-b/vscode-duktape-debug/issues/63